vtex/admin-ui
# Combobox mult object

#### What is the purpose of this pull request?
Adding custom rendering to comboboxmultiple and add docs for custom rendering.

# Breaking changes:

 - **useCombobox returns matches and setMatches now, these should be used to control the rendered list, not "list" anymore.**

- for useComboboxMultiple state **removeSelected was renamed to unselect**

- for useComboboxMultiple state **selected and setSelected have been removed**, to manage the selected items you should use `selectedItems`, and `setSelectedItems`.

#### How should this be manually tested?

Check out the built docs, the built storybook, check the code examples and see if they make sense.

#### Screenshots or example usage
<img width="394" alt="image" src="https://user-images.githubusercontent.com/8623116/162053968-80340b57-d727-42e9-a2b3-9e2550ff608a.png">


#### Types of changes
